### ISSUE TITLE
Changing swipe sensitivity

### ISSUE THREAD

**Issue by jonloh** [OwlCarousel2/OwlCarousel2#629]
Is there another way to adjust the swipe sensitivity? Currently when you just swipe just a little bit, it will trigger to switch to the next item.

**Comment by mrjosa** [comment]
Yes, this would be very good. It is almost not usable on a mobile device as it is now.

**Comment by MTShare** [comment]
+1

**Comment by greg5green** [comment]
It's written to switch items if you swipe at all, so this is not a bug (and is quite usable in my testing! :smile: ).

This isn't a high priority thing to be working on, but we do have a todo in the code ([line 692](https://github.com/smashingboxes/OwlCarousel2/blob/a363380b2839ab15e75772b59a16352a11a78333/src/js/owl.carousel.js#L692)) to add this feature. If anyone has an interest in adding this feature and submitting a PR, please speak up and get to work :D
